README.md

```diff
@@ -74,25 +74,34 @@ The [**MSSQL Extension for Visual Studio Code**](https://www.aka.ms/vscode-mssql
     -   Save comparisons to rerun or audit schema changes
--   **Customizable Extension Options**: Configure command shortcuts, appearance, and other settings to personalize your development experience.
 -   **GitHub Copilot Integration**: Boost your productivity with AI‑assisted SQL development
-    -   Intelligent T‑SQL completions and natural‑language prompts
-    -   Explain objects and generate schema or test data
-    -   Identify risky patterns and improve SQL security practices
--   **GitHub Copilot Agent Mode** Let Copilot perform database tasks on your behalf
+    -   Chat with `@mssql` using natural language to generate queries, explain stored procedures, scaffold schemas, and debug SQL issues with database-aware context
+    -   Intelligent code completions and inline suggestions while coding, with support for popular ORMs and T-SQL
+    -   Query optimization with AI recommendations to refactor slow queries, fine-tune indexes, and understand execution plans
+    -   Generate mock and test data automatically with sample data and seeding scripts
+    -   Identify risky patterns such as SQL injection and over-permissive roles, with suggestions for safer alternatives
+    -   Explain complex stored procedures, views, and functions in plain language—perfect for onboarding and code reviews
+-   **GitHub Copilot Agent Mode**: Let Copilot perform database tasks on your behalf
     -   Securely executes actions like connecting, switching databases, or running queries directly from chat
     -   Surfaces schema details and connection info without manual navigation
     -   Provides a confirmable, AI-driven assistant for common database workflows
     -   Access all approved Agent tools from the Tools panel
 -   **GitHub Copilot Slash Commands**: Quick, discoverable shortcuts in chat
-    -   Type `/` to see commands like `/connect`, `/changeDatabase`, `/runQuery`, `/explain`, `/fix`, `/optimize`
+    -   Type `/` to see commands like `/connect`, `/changeDatabase`, `/runQuery`, `/explain`, `/fix`, `/optimize`, and more
     -   Connection commands open the MSSQL connection panel; query commands accept input and return results in chat
+-   **Customizable Extension Options**: Configure command shortcuts, appearance, and other settings to personalize your development experience.
 
 ## Public Preview Features
 
 -   **Fabric Integration (`Preview`)**: Browse workspaces and provision SQL databases in Fabric directly from VS Code
     -   Sign in with Microsoft Entra ID, browse workspaces, search, and connect to SQL databases or SQL analytics endpoints from the Connection dialog (includes **Open in MSSQL** from the Fabric extension)
     -   Create a SQL database from the Deployments page; capacity‑aware and **auto‑connects** when complete
+-   **View & Edit Data (`Preview`)**: Browse and modify table data directly within the editor without writing Transact-SQL data manipulation language (DML) statements
```

Comment:
To confirm, no mention of the dacpac dialog because it's not "public", right?

Reply:
I completely missed that. Thanks for the reminder, the content was added now.